plugins/database/src/low_level_odbc.cpp

```diff
@@ -315,15 +313,13 @@ cllCheckPending( const char *sql, int option, int dbType ) {
             }
         }
 
-        rodsLog( LOG_NOTICE, "Warning, pending SQL at cllDisconnect, count: %d",
-                 pendingCount );
+        log_db::trace("Warning, pending SQL at cllDisconnect, count: %d", pendingCount);
         int max = maxPendingToRecord;
         if ( pendingIx < max ) {
             max = pendingIx;
         }
         for ( int i = 0; i < max; i++ ) {
-            rodsLog( LOG_NOTICE, "Warning, pending SQL: %s ...",
-                     ( char * )&pBuffer[i * pendingRecordSize] );
+            log_db::trace("Warning, pending SQL: %s ...", (char*) &pBuffer[i * pendingRecordSize]);
```

Comment:
clang-tidy says this is a redundant cast. Plus, it's a C-style cast. Let's get rid of it

Reply:
Done. Added ignore for other errors on the line.